improve template customisation documentation (#1821)
nf-core/tools: README.md

  - nf_core_configs
```

This will create a pipeline called `coolpipe` in the directory `cool-pipes-company-coolpipe` (`<prefix>-<name>`) with `me` as the author. It will exclude all possible parts of the template:

> - `github`: removed all files required for GitHub hosting of the pipeline. Specifically the `.github` folder and `.gitignore` file.
> - `ci`: removes the GitHub continuous integration tests from the pipeline. Specifically the `.github/workflows/` folder.

Reviewer: ```suggestion
- `github`: removed all files required for GitHub hosting of the pipeline. Specifically, the `.github` folder and `.gitignore` file.
- `ci`: removes the GitHub continuous integration tests from the pipeline. Specifically, the `.github/workflows/` folder.
```

The `nf-core create` command comes with a number of options that allow you to customize the creation of a pipeline if you intend to not publish it as an
nf-core pipeline. This can be done in two ways: by using interactive prompts, or by supplying a `template.yml` file using the `--template-yaml <file>` option.
Both options allow you to specify a custom pipeline prefix to use instead of the common `nf-core`, as well as selecting parts of the template to be excluded during pipeline creation.
The interactive prompts will guide you through the pipeline creation process. An example of a `template.yml` file is shown below.

```yaml
name: coolpipe
description: A cool pipeline
author: me

> prefix: cool-pipes-company

Author: ```suggestion
prefix: myorg
```

  - nf_core_configs
```

> This will create a pipeline called `coolpipe` in the directory `cool-pipes-company-coolpipe` (`<prefix>-<name>`) with `me` as the author. It will exclude all possible parts of the template:

Author: ```suggestion
This will create a pipeline called `coolpipe` in the directory `myorg-coolpipe` (`<prefix>-<name>`) with `me` as the author. It will exclude all possible parts of the template:
```
